scheduler/src/cook/kubernetes/controller.clj

```diff
@@ -280,7 +281,7 @@
         (log/info "In compute cluster" name ", encountered OutOfcpu pod status reason for" instance-id)
         :reason-invalid-offers)
 
-      (api/pod-unschedulable? instance-id pod-status)
+      (api/pod-unschedulable? instance-id pod-status pod)
```

Comment:
Worth a comment that this needs to know the pod because we have different timeouts for different depending on if we binpack via fenzo or if k8s binpacks, with a TODO that we won't need this when we're out of this hybrid state.

Reply:
Done